### ISSUE TITLE
gradle repository problem for version 1.4.3

### ISSUE THREAD

**Issue by duddns** [unbroken-dome/gradle-testsets-plugin#43]
Hello,
There was a problem with the gradle repository.

gradle build error
```
Could not find gradle.plugin.org.unbroken-dome.gradle-plugins:gradle-testsets-
plugin:1.4.3.
```

and link error
https://plugins-artifacts.gradle.org/gradle.plugin.org.unbroken-dome.gradle-plugins/gradle-testsets-plugin/1.4.3/e1dbaadf8068147ef7f312fc722ec971f52465b502f430a995ef0d8ba093576f/gradle-testsets-plugin-1.4.3.pom
```
<Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message><Key>gradle.plugin.org.unbroken-dome.gradle-plugins/gradle-testsets-plugin/1.4.3/e1dbaadf8068147ef7f312fc722ec971f52465b502f430a995ef0d8ba093576f/gradle-testsets-plugin-1.4.3.pom</Key><RequestId>48D2CF5BE03DD308</RequestId><HostId>FPsIZBIHlePZQORZoBNrYGOAKfPboeMu0a5eB8lMPf2RPYPqAjcpeX88FdOvn/6WWJUIoWpeq7k=</HostId></Error>
```

Regards.

**Comment by duddns** [comment]
Looks like another problem.
It works fine now.

**Comment by ketan** [comment]
This is still a problem for me. I had to use the following to make it work:

```
buildscript {
    repositories {
        maven {
            url "https://plugins.gradle.org/m2/"
        }
    }

    dependencies {
        classpath "org.unbroken-dome.gradle-plugins:gradle-testsets-plugin:1.4.3"
    }
}

apply plugin: "org.unbroken-dome.test-sets"
```

**Comment by tkrullmann** [comment]
It seems there was a problem with the publication of the plugin in v1.4.3.
I've published 1.4.4, there are no functional changes, I hope that it works better.
